cfgov/scripts/fix_links_in_tableblocks.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import get_object_or_404
+from bs4 import BeautifulSoup
+
+from wagtail.wagtaildocs.models import get_document_model
+from wagtail.wagtailcore.models import Page
+
+from v1.models.base import CFGOVPage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+def get_tableblocks(page):
+    """ Get all the TableBlocks for a given page.
+    TableBlocks can be stored either directly in page's content,
+    or in a FullWidthText item. So we must check both.
+    """
+    try:
+        stream_data = page.specific.content.stream_data
+    except:
+        return []
+    tableblocks = list(filter(lambda item: item['type'] == 'table_block', stream_data))
+    full_width_text_items = list(filter(lambda item: item['type'] == 'full_width_text', stream_data))
+    for item in full_width_text_items:
+        sub_items = item['value']
+        for sub_item in list(filter(lambda i: i['type'] == 'table_block', sub_items)):
+            tableblocks.append(sub_item)
+    return tableblocks
+
+
+def convert_links(links):
+    """ Adds a href to the link with the reative path if a
+    document ID or page ID is stored
+    """
+    updated = False
+    for link in links:
+        linktype = link.get('linktype')
+        object_id = link.get('id')
+        if link.get('href'):
+            continue  # Don't update it if the href is already set
+        if linktype == 'document':
+            doc = get_object_or_404(get_document_model(), id=object_id)
+            filepath = '/' + doc.file.name
```

Comment:
Converted document links don't resolve properly for me. When I run this script on page [1424](http://localhost:8000/policy-compliance/enforcement/actions/integrity-advance/), the link

`<a id="1033" linktype="document">176</a>`

gets converted to

`<a href="/documents/2015-CFPB-0029_Document_176_09272016.pdf" id="1033" linktype="document">176</a>`

That URL doesn't resolve for me. From looking at the [urlconf](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/urls.py#L33) I think this is missing the ID, e.g. it should be

`<a href="/documents/1033/2015-CFPB-0029_Document_176_09272016.pdf" id="1033" linktype="document">176</a>`

Instead of using `doc.file.name`, could this use `d.url` (with no need for a leading slash)?

```py
>>> from wagtail.wagtaildocs.models import Document
>>> d = Document.objects.get(pk=1033)
>>> d.url
u'/documents/1033/2015-CFPB-0029_Document_176_09272016.pdf'
```